vic/drivers/shared_image/src/vic_init.c

```diff
@@ -792,6 +796,120 @@ vic_init(void)
 
         soil_moisture_from_water_table(&(soil_con[i]), options.Nlayer);
 
+        // Soil thermal node thicknesses and positions
```

Comment:
There's already a check for that in vic_restore.c, line 1012 - a mismatch raises an error.